dma_frontend/src/components/SignInComp.jsx

```diff
@@ -1,11 +1,69 @@
 import React from 'react';
-import '../styles/login.css';
-
+import '../styles/signin.css';
+import signin from '../images/signin.jpg'
 const SignInComp = () => {
   return (
     <div className=''>
       <div className=''>
-        {/* TYPE YOUR CODE HERE. IF YOU USE TAILWINDCSS MAKE SURE TO ADD THE tw- PREFIX TO THE UTILITY CLASS AND IF YOU WRITE VANILLA CSS KINDLY USE login.css FILE */}
+        {         <div class="in-container">
+         <div class="in-item-2">
+           <img id="flex-img-in" src={signin} alt="img2" />
+         </div>
+         <div class="in-item-1">
+           <div class="known-in">
+             <div class="tnav-in">
+               <img src="/svg/svg" alt="" />
+               <div id="myLinks">
+                 <a href="HomePage.jsx">Home</a>
+                 <a href="SignInComp.jsx">Sign In</a>
+                 <a href="ContactUsComp.jsx">Contact Us</a>
+               </div>
+               {/* <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                 <i class="fa fa-bars"></i>
+               </a> */}
+             </div>
+           </div>
+           <img id="img-desktop-in" src="/svg/svg" alt="" />
+           <p id="p-in">Sign In</p>
+ 
+           <form class="forme">
+ 
+             <label for="email">Email address*</label><br /><br />
+             <input
```

Comment:
Please check through this whole file and change any label entity that has `for` to `htmlFor`.  

I only highlighted this section as an example for you. please look through the whole file and change anywhere this kind of issue appears